common/model-views.cpp

```diff
@@ -771,13 +774,9 @@ namespace rs2
                             // computing the number of decimal digits taken from the step options' property
                             // this will then be used to format the displayed value
                             auto num_of_decimal_digits = [](float f) {
-                                int res = 0;
-                                while (f && (int)f == 0)
-                                {
-                                    f *= 10.f;
-                                    ++res;
-                                }
-                                return res;
+                                float f_0 = std::abs(f - (int)f);
+                                std::string s = std::to_string(f_0);
+                                return s.length() - 2;
```

Comment:
Check for case of 1/0 decimal digits.
switch to fabs

Reply:
Done